dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/PowerSystemStabilizers/Standard/PssIEEE2B.mo

```diff
@@ -19,10 +19,11 @@ model PssIEEE2B "IEEE Power System Stabilizer type 2B"
   import Dynawo.Types;
```

Comment:
Why some changes in this model ?

Reply:
Fraunhofer has suggested to enable several input possibilities for the omega branch (speed, speed deviation) and it will be easier in a preassembled.